src/apurva/Mall.java

```diff
@@ -0,0 +1,50 @@
+package apurva;
+
+class Mall {
+	void itemAddedToCart(int items) {
+		System.out.println(items);
+	}
+
+	void customerName(String name) {
+		System.out.println(name);
+	}
+
+	void paymentDoneUsing(String visaCard, String cash) {
+		System.out.println();
+	}
+
+	void displayCustomerInfo(int items, String visaCard, String cash){
+		String payment = null;
+		if (items<10 && payment==cash)
+			System.out.println();
+		else if (items>10 && payment==visaCard)
+			System.out.println();
+		else if (items>10 && payment==cash)
+			System.out.println();
+		else if (items<10 && payment==visaCard)
+			System.out.println();
+	}
+
+	void needBill(boolean bill) {
+		System.out.println(true);
```

Comment:
Here, by printing true, how customer will know about true! same for haveCarryBag()